ci/test-conformance-gke.sh

```diff
@@ -0,0 +1,256 @@
+
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --cluster-name)
+    CLUSTER="$2"
+    shift 2
+    ;;
+    --gke-project)
+    GKE_PROJECT="$2"
+    shift 2
+    ;;
+    --svc-account)
+    SVC_ACCOUNT_NAME="$2"
+    shift 2
+    ;;
+    --user)
+    USER_EMAIL="$2"
+    shift 2
+    ;;
+    --gke-zone)
+    GKE_ZONE="$2"
+    shift 2
+    ;;
+    --svc-cidr)
+    GKE_SERVICE_CIDR="$2"
+    shift 2
+    ;;
+    --host-type)
+    GKE_HOST="$2"
+    shift 2
+    ;;
+    --k8s-version)
+    K8S_VERSION="$2"
+    shift 2
+    ;;
+    --setup-only)
+    RUN_SETUP_ONLY=true
+    RUN_ALL=false
+    shift
+    ;;
+    --cleanup-only)
+    RUN_CLEANUP_ONLY=true
+    RUN_ALL=false
+    shift
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+function setup_gke() {
+
+    if [[ -z ${CLUSTER+x} ]]; then
+        CLUSTER="${JOB_NAME}-${BUILD_NUMBER}"
+    fi
+    if [[ -z ${K8S_VERSION+x} ]]; then
+        K8S_VERSION="1.15.11-gke.11"
+    fi
+
+    echo "=== This cluster to be created is named: ${CLUSTER} ==="
+    echo "CLUSTERNAME=${CLUSTER}" > ci_properties.txt
+
+    echo "=== Using the following gcloud version ==="
+    gcloud --version
+    echo "=== Using the following kubectl ==="
+    which kubectl
+
+    echo '=== Creating a cluster in GKE ==='
+    gcloud container --project ${GKE_PROJECT} clusters create ${CLUSTER} --image-type ${GKE_HOST} \
+                     --cluster-version ${K8S_VERSION} --zone ${GKE_ZONE} \
+                     --enable-ip-alias --services-ipv4-cidr ${GKE_SERVICE_CIDR}
+    if [[ $? -ne 0 ]]; then
+        echo "=== Failed to deploy GKE cluster! ==="
+        exit 1
+    fi
+
+    sleep 10
+    if [[ $(kubectl get nodes) ]]; then
+        echo "=== GKE cluster setup succeeded ==="
+    else
+        echo "=== GKE cluster is not configured correctly! ==="
+        exit 1
+    fi
+}
+
+function deliver_antrea_to_gke() {
+
+    echo "=== Configuring Antrea for cluster ==="
+
+    if [[ -z ${SVC_ACCOUNT_NAME+x} ]]; then
+        gcloud projects add-iam-policy-binding $GKE_PROJECT --member serviceAccount:${SVC_ACCOUNT_NAME} --role roles/container.admin
+        kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user ${SVC_ACCOUNT_NAME}
+    elif [[ -z ${USER_EMAIL+x} ]]; then
+        gcloud projects add-iam-policy-binding $GKE_PROJECT --member user:${USER_EMAIL} --role roles/container.admin
+        kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user ${USER_EMAIL}
+    else
+        echo "Neither service account or user email info is set, cannot create cluster-admin-binding!"
+             "Please refer to --help for more information."
```

Comment:
Does that actually work? I think we need to repeat the `echo` statement

Reply:
Thanks for the catch. Done